mysql-test/mroonga/storage/collation/utf8mb4_uca1400_ai_ci/r/japanese.result

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS diaries;
+SET NAMES utf8mb4;
+CREATE TABLE diaries (
+content varchar(256) COLLATE utf8mb4_uca1400_ai_ci,
+FULLTEXT INDEX (content)
+) DEFAULT CHARSET=utf8mb4;
+INSERT INTO diaries VALUES ("はひふへほ");
+INSERT INTO diaries VALUES ("ばびぶべぼ");
+INSERT INTO diaries VALUES ("ハヒフヘホ");
+INSERT INTO diaries VALUES ("バビブベボ");
+INSERT INTO diaries VALUES ("ﾊﾋﾌﾍﾎ");
+INSERT INTO diaries VALUES ("ﾊﾞﾋﾞﾌﾞﾍﾞﾎﾞ");
```

Comment:
Could you and any record that doesn't match?

Reply:
fix: [8185a2d](https://github.com/mroonga/mroonga/pull/908/commits/8185a2da7302420253a5a2b43c521f728f1342b7) I added the unmatched cases.